docs/install.rst

```diff
@@ -117,4 +157,16 @@ Testing the Installation
 After installation, you can run both the compiled and python unit tests.
 These tests will create an output directory in your current working directory::
 
-    %> python -c "import toast.tests; toast.tests.run()"
```

Comment:
This python command is still the way to run the unit tests.  It runs the python tests as well as the compiled tests.  The `toast_test` executable only runs the compiled tests.

Reply:
Done, I have reverted the old line of code.